packages/api/src/routes/medical/schemas/fhir.ts

```diff
@@ -0,0 +1,18 @@
+import { z } from "zod";
+
+const typeSchema = z.enum(["collection"]);
+
+const bundleEntrySchema = z.array(
+  z.object({
+    resource: z.any(),
```

Comment:
NABD, but we could validate it has `resourceType` which is required by the `validateFhirEntries` function.

Reply:
So i did 
```
resource: z.object({
    resourceType: z.string()
}),
```
But then it would only return the resoureType after parsing

Reply:
Could try `.passthrough()`: https://zod.dev/?id=passthrough

Reply:
It doesnt match the Bundle Resource because it cant match an object with just resourceType

Reply:
That's weird. The goal of passthrough is to be able to define a prop on the schema and have the object being validated having that prop plus others that are not validated.

I'd double check, just to make sure we're testing it correctly. NABD otherwise.

Reply:
<img width="631" alt="Screenshot 2023-07-27 at 7 51 51 PM" src="https://github.com/metriport/metriport/assets/36802074/5fda617d-0158-4e4b-95d6-027bdd416b60">
<img width="784" alt="Screenshot 2023-07-27 at 7 52 24 PM" src="https://github.com/metriport/metriport/assets/36802074/ed983f92-089c-4749-b350-1c4098ca9a4f">